src/trivia/util.h

```diff
@@ -643,6 +643,13 @@ is_exp_of_two(unsigned n)
 void
 thread_sleep(double sec);
 
+/**
+ * Getenv wrapper returning the corresponding value only when it'd fit in a
+ * buffer of the given size.
+ */
+char *
+getenv_safe(const char *name, size_t buf_size, bool *exceeded);
```

Comment:
It would be nice to clarify a corner case: if the variable has exactly `buf_size` length, it leads to `exceeded == true`, because there is no place to store leading `'\0'`.

Reply:
Sure:
```diff
diff --git a/src/trivia/util.h b/src/trivia/util.h
index c2683aafd..311c898e3 100644
--- a/src/trivia/util.h
+++ b/src/trivia/util.h
@@ -645,7 +645,8 @@ thread_sleep(double sec);
 
 /**
  * Getenv wrapper returning the corresponding value only when it'd fit in a
- * buffer of the given size.
+ * buffer of the given size. That is, its length should be <= buffer size - 1,
+ * because one extra char is taken by the trailing '\0'.
  */
 char *
 getenv_safe(const char *name, size_t buf_size, bool *exceeded);
```